dataikuapi/dss/ml.py

```diff
@@ -544,7 +552,24 @@ def validation_mode(self, mode):
         assert mode in {"KFOLD", "SHUFFLE", "TIME_SERIES_KFOLD", "TIME_SERIES_SINGLE_SPLIT", "CUSTOM"}
         self._raw_settings["mode"] = mode
 
-    def set_kfold_validation(self, n_folds=5, stratified=True):
+    @property
+    def cv_seed(self):
+        """
+        :return: cross-validation seed for splitting the data during hyperparameter search
+        :rtype: int
+        """
+        return self._raw_settings["cvSeed"]
+
+    @cv_seed.setter
+    def cv_seed(self, seed):
+        """
+        :param seed: cross-validation seed for splitting the data during hyperparameter search
+        :type seed: int
+        """
+        assert isinstance(seed, int)
+        self._raw_settings["cvSeed"] = seed
+
+    def set_kfold_validation(self, n_folds=5, stratified=True, cv_seed=0):
```

Comment:
Where does the default value `0` come from ?

Reply:
It's the default `int` value.

Reply:
It's arbitrary.